.github/workflows/ci.yml

```diff
@@ -219,11 +219,12 @@ jobs:
         env:
           TEMP: "R:\\Temp"
 
+  # TODO: Remove this when we add Python 3.11 to CI.
```

Comment:
We can do this now FWIW, `3.11.0-alpha - 3.11`?

Reply:
Let me try.

Reply:
Ah, it’s a bit difficult to convert `3.11.0-alpha.7` to `3.11` for `nox -s test-{{ python }}`…

Reply:
You can use the include field to add an additional key-value to the `3.11.0-alpha.7` entry -> https://github.com/cooperlees/black-primer/blob/6c40328df33f72468105466423edd2115be491ff/.github/workflows/ci.yaml#L30-L35 and then use a OR expression -> https://github.com/cooperlees/black-primer/blob/6c40328df33f72468105466423edd2115be491ff/.github/workflows/ci.yaml#L53

Reply:
Ah, this is the syntax I was looking for, thanks

```
${{ matrix.nox-python || matrix.python }}
```

Reply:
Seem like it’s not ready for virtualenv (which we need for Nox) unfortunately

```
ERROR: The executable /home/runner/work/pip/pip/.nox/test-3-11/bin/python3.11 is not functioning
ERROR: It thinks sys.prefix is '/opt/hostedtoolcache/Python/3.11.0-alpha.7/x64' (should be '/home/runner/work/pip/pip/.nox/test-3-11')
ERROR: virtualenv is not compatible with this system or executable
Running virtualenv with interpreter /opt/hostedtoolcache/Python/3.11.0-alpha.7/x64/bin/python3.11
```

Reply:
Yea, we're stuck on virtualenv 16.x for our test suite. That's now a proper problem. :)